hadoop-hdds/common/pom.xml

```diff
@@ -206,6 +206,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-platform-launcher</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-interface-server</artifactId>
+    </dependency>
```

Comment:
Adding dependency in `hdds-common` on the server-only interface brings it to the client-side, which makes the separation of `hdds-interface-*` useless.

Reply:
Noted, I will try removing the dependency as suggested. 

I was working on accessing the methods from the `ScmBlockLocationProtocolProtos` class (package: `org.apache.hadoop.hdds.protocol.proto`) located under the package: `org.apache.hadoop.hdds.scm.net` for the purpose of introducing the _from and to protobuf_ methods for the newly added proto objects.

Reply:
Instead of code refactoring, seeing if shifting the newly introduced proto message definitions to `ScmAdminProtocol.proto` (`StorageContainerLocationProtocolProtos`) would avoid unnecessary dependency inclusions.

Reply:
I have moved the newer proto objects to `hdds.proto` instead due to a few implementation constraints and retained the request, response and type proto definitions within `ScmServerProtocol.proto`, thanks!